src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -114,3 +118,120 @@ export const schema = {
     },
   },
 };
+
+const indexOfChanged = (oldArr, newArr) => {
```

Comment:
recommend an explanatory comment here around assumption that only one element will ever be different, or maybe renaming the function to something like `indexOfFirstChange`